src/main/java/com/danliuk/service/BusTicketService.java

```diff
@@ -0,0 +1,42 @@
+package com.danliuk.service;
+
+import com.danliuk.model.BusTicket;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BusTicketService {
+    public static List<BusTicket> getObjectsFromFile(String filePath) throws IOException {
+        List<String> lines;
```

Comment:
to avoid NullPointerException initialize the value 
List<String> jsonTickets = new ArrayList<>();